File: dart/variables/3.dart
import 'dart:io';

void main() {
  print("Calcular distancia de movimiento rectilineo");

  print("Digite la velocidad en km/h: ");

  int vel = int.parse(stdin.readLineSync()!);
  
  print("Digite el tiempo en horas: ");
  
    int tiem = int.parse(stdin.readLineSync()!);

  int dis = vel * tiem;

  print("La distancia recorrida es igual a ${dis} Km");
  
}

File: variablesCsharp/3.cs
using System;

namespace Variables
{
    internal class Tres
    {
        public Tres() {
            Console.WriteLine("Digite la velocidad en Km/h: ");
            int velocidad = int.Parse(Console.ReadLine());

            Console.WriteLine("Digite el tiempo en horas: ");
            int tiempo = int.Parse(Console.ReadLine());

            int distancia = velocidad * tiempo;

            Console.WriteLine("El resultado de X es igual a " + distancia + "Km");
        }
    }
}
